apps/browser/qml/pages/SettingsPage.qml

```diff
@@ -41,21 +47,32 @@ Page {
                 active: !AccessPolicy.browserEnabled
             }
 
-            TextField {
+            BrowserListItem {
                 id: homePage
-                enabled: AccessPolicy.browserEnabled
 
                 //: Label for home page text field
                 //% "Home Page"
                 label: qsTrId("settings_browser-la-home_page")
-                text: homePageConfig.value == "about:blank" ? "" : homePageConfig.value
-
-                inputMethodHints: Qt.ImhNoPredictiveText | Qt.ImhNoAutoUppercase | Qt.ImhUrlCharactersOnly
-
-                onTextChanged: homePageConfig.value = text || "about:blank"
-
-                EnterKey.iconSource: "image://theme/icon-m-enter-close"
-                EnterKey.onClicked: focus = false
+                //% "Default"
+                value: homePageConfig.value === "about:blank" ? qsTrId("sailfish_browser-la-home_page_default") : removeProtocolTypeFromUri(homePageConfig.value)
+                iconSource: "image://theme/icon-m-home"
+                menu: ContextMenu {
+                    MenuItem {
+                        //% "Default home page"
+                        text: qsTrId("sailfish_browser-me-home_page_default")
+                        onClicked: homePageConfig.value = "about:blank"
+                    }
+                    MenuItem {
+                        readonly property string site: removeProtocolTypeFromUri(homePageConfig.value)
+                        //% "Custom website %1"
```

Comment:
It might be helpful for the translators if you could explain what the `%1` will be replaced with.

Reply:
@llewelld Done